### PR TITLE
Modify FindCommand such that it searches from both name and company name 

### LINKED ISSUE #68: Modify FindCommand feature such that it searches both 'Name' and 'Company Name'

### PR DESCRIPTION
**Pull request by ashleyy2444**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/NameOrCompanyNameContainsKeywordsPredicate.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} or {@code CompanyName} matches any of the keywords given.
+ */
+public class NameOrCompanyNameContainsKeywordsPredicate implements Predicate<Person> {
```
Comment (dabzpengu) [inline src/main/java/seedu/address/model/person/NameOrCompanyNameContainsKeywordsPredicate.java:12]: nice naming of the class!

**Review by Lalelulilulela — APPROVED** [review]
LGTM :)

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.logic.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.NameOrCompanyNameContainsKeywordsPredicate;
 
 /**
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
```
Comment (Lalelulilulela) [inline src/main/java/seedu/address/logic/commands/FindCommand.java]: Can update the header comments

**Review by Lalelulilulela — APPROVED** [review]
LGTM

**Pull request merged**